.github/workflows/Tests-arm64.yaml

```diff
@@ -0,0 +1,56 @@
+name: Tests-arm64
+on:
+  schedule:
+    - cron: '30 2 * * *' # runs daily at 1:30 am.
```

Comment:
As mentioned previously, please update these to `on: pull_request` so we can verify the workflows before merging.